build0.sh

```diff
@@ -16,7 +16,7 @@ python grammar/asdl_cpp.py src/libasr/ASR.asdl src/libasr/asr.h
 python src/libasr/wasm_instructions_visitor.py
 
 # Generate the tokenizer and parser
-(cd src/lpython/parser && re2c -W -b tokenizer.re -o tokenizer.cpp)
+(cd src/lpython/parser && re2c -W -b tokenizer.re -o tokenizer.cpp --case-inverted)
```

Comment:
Let's not use the `--case-inverted`, because I think it will become confusing with LFortran.

I think all we need is to change `'raise'` to `"raise"` and the same for all the other keywords.

Reply:
Yup, its works. Thank you!